### ISSUE TITLE
Obfuscar doesn't obfuscating anything

### ISSUE THREAD

**Issue by lextudio-support** [obfuscar/obfuscar#502]
Greetings! I tried obfuscar with simple configuration file:

```
<?xml version='1.0'?>
<Obfuscator>
  <Var name="InPath" value=".\Obfuscator_Input" />
  <Var name="OutPath" value=".\Obfuscator_Output" />

  <Module file="$(InPath)\WindowsFormsApplication1.exe" /> 
</Obfuscator>
```
But when i opened result .exe with ILspy i saw that the code is still the same, except that some functions name was changed to "A". Application is created with VS2013 and .NET 4.5. What am i doing wrong? Any help or advice would be appreciated. Thank you!

**Original Reported Date**: 2015-07-23T23:58:39.573-07:00
**Planned For Release**: 
**Original CodePlex ID**: 27

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

https://obfuscar.codeplex.com/wikipage?title=Configuration

Please take a look at all the options.

**Original Posted Date**: 2015-08-04T05:42:57.01-07:00

**Comment by lextudio-support** [comment]
**Original Closed Date**: 2015-08-04T05:43:03.39-07:00
